network/src/lib.rs

```diff
@@ -0,0 +1,132 @@
+pub mod client;
+pub mod constants;
+pub mod http;
+pub mod server;
+
```

Comment:
IMHO it will be great to have all `pub` entities here. So users will just do:
```
use iroha_network::prelude::*;

let network = Network::new(...);
...
network.send_request(Request::from_bytes(query.into()))?;
```